spec/marc_to_solr/lib/config_spec.rb

```diff
@@ -4,862 +4,874 @@
   let(:leader) { '1234567890' }
   let(:online) { @indexer.map_record(fixture_record('9990889283506421')) }
 
-  def fixture_record(fixture_name)
+  def fixture_record(fixture_name, indexer: @indexer)
     f = File.expand_path("../../../fixtures/marc_to_solr/#{fixture_name}.mrx", __FILE__)
-    @indexer.reader!(f).first
```

Comment:
I created a context for valid records, so all of these lines are still part of the file.  They were just indented.  The diff looks much larger because of this.  This allowed me to test my invalid record without having to load all the valid ones.